src/test/java/racingcar/car/CarTest.java

```diff
@@ -90,4 +90,67 @@ void simpleRacingCar() {
 
         assertThat(cars.get(0).getPosition()).isNotEqualTo(cars.get(1).getPosition());
     }
+
+    @ParameterizedTest
+    @ValueSource(strings = {"봉봉봉봉봉봉카", "봉봉봉일균카", "넥스트스탭카"})
+    @DisplayName("자동차명은 5자이하허용")
+    void carNames(String carName) {
+        assertThatThrownBy(() -> {
+            Car car = new NextStepCar(carName);
+        }).isExactlyInstanceOf(IllegalArgumentException.class);
+    }
+
+    @ParameterizedTest
+    @ValueSource(strings = {"봉카,봉일균카,넥스트카"})
+    @DisplayName("이름을 입력받다 - 쉼표로 구분")
+    void splitByCommas(String carNames) {
+        List<Car> carList = new ArrayList<>();
+        for(String carName: carNames.split(",")) {
+            carList.add(new NextStepCar(carName));
+        }
+        NextStepCars nextStepCars = new NextStepCars(carList);
+        assertThat(nextStepCars.carCount()).isEqualTo(3);
+    }
+
+    @Test
+    @DisplayName("우승자찾기 단건")
+    void winnerOnlyOne() {
+        String carNames = "봉,봉봉,봉봉봉,봉봉봉봉";
+        List<Car> carList = new ArrayList<>();
+        int moveIndex = 1;
+
+        for(String carName: carNames.split(",")) {
+            NextStepCar car = new NextStepCar(carName);
+            car.moveForward(moveIndex++);
+            carList.add(car);
+        }
+        NextStepCars nextStepCars = new NextStepCars(carList);
+        assertThat(nextStepCars.getWinner().get(0)).isEqualTo("봉봉봉봉");
```

Comment:
getter를 통해 값을 가져오는 것 보다 논리적인 동치성을 비교할 수 있도록 좀 더 객체지향스럽게 리팩토링 해보면 좋을 것 같아요!
다음과 같이 테스트를 해볼 수 있도록 변경해볼 수 있을까요 :) ?
```suggestion
        final Car expect = new NextStepCar("봉봉봉봉");
        expect.moveForward(4);
        
        assertThat(nextStepCars.getWinner()).isEqualTo(expect);
```